website/_layouts/home.html

```diff
@@ -21,20 +21,21 @@ <h4>Watch it live on the <a href="https://twitch.tv/therubygalaxy">Ruby Galaxy</
          {% include host_header.html hosts=event.hosts %}
          <h5>{{ event.description }}</h5>
       </div>
-      <!-- For future speaker bios and talk abstracts -->
-      <!-- <div class="column">
-         {% assign event = site.data.events.last %}
-         <h3>Talks for {{ event.event_text }}</h3>
-         <br>
-         {% for talk in event.talks %}        
-            <div class="column">        
-               <h4>{{ talk.name }}</h4> 
-               <p>{{ talk.abstract }}</p><br>
-               <small>{{ talk.author }}</small> <a href="{{ talk.author_twitter }}" style="color:#f93434"><img src="/assets/img/twitter-brands.svg" style="height: 15px; width: 30px; display: inline-block; fill:#f93434" /></a><br>
-               <small>{{ talk.author_bio }}</small><br>
-            </div> 
-         {% endfor %}
-      </div> -->
+      <!-- For speaker bios and talk abstracts -->
+      {% if event.talks.count > 0 %}
```

Comment:
Now, instead of commented out code this conditional allows for the talks to be updated on the page once added to the events.yml file.

Reply:
@ChaelCodes Added!

Reply:
Did you test this? Did it work? I ask because we assign event on line 27.

Reply:
It did work but youre right. Once I tested it with an event with talks it didn't work. One sec!

Reply:
Oh wowww spent way to much time with `count` but I guess you cant use count in liquid? So I changed it to `{% if event.talks != nil %}`